dashboard/test/lib/pd/payment/payment_calculator_csf_test.rb

```diff
@@ -12,13 +12,15 @@ class PaymentCalculatorCSFTest < ActiveSupport::TestCase
         create :user_level, user: @qualified_teacher, best_result: ::ActivityConstants::MINIMUM_PASS_RESULT
       end
       @workshop.section.add_student @qualified_teacher
+      create :pd_enrollment, workshop: @workshop, user: @qualified_teacher
 
       # < 10 passing levels: unqualified
       @unqualified_teacher = create :teacher
       9.times do
         create :user_level, user: @unqualified_teacher, best_result: ::ActivityConstants::MINIMUM_PASS_RESULT
       end
       @workshop.section.add_student @unqualified_teacher
+      create :pd_enrollment, workshop: @workshop, user: @unqualified_teacher
```

Comment:
This is just for CSF so I don't think it makes sense to generalize.
All enrollments are qualified by default, except for CSF. 

In this case, the qualification has to do with puzzles the associated teacher has solved independent of the enrollment so again I don't think that factory is the best place for it.